specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/computeRPCommon.json

```diff
@@ -1335,8 +1336,9 @@
           "description": "[Preview Feature] Specifies settings related to VM Guest Patching on Linux."
         },
         "enableVMAgentPlatformUpdates": {
+          "readOnly": true,
```

Comment:
>"readOnly": true, [](http://example.com/codeflow?start=10&length=17)

This is a potentially severe breaking change for Azure Policy customers. This property has been onboarded to Azure Policy as modifiable, which means that customers may have deployed governance policies that attempt to automatically modify it in the ARM frontdoor during a PUT. When that fails, that customer can be completely unable to perform PUTs to the affected resource types.

You need to engage with the Azure Policy team to understand the impact of this change before rolling it out. Start with policypm@microsoft.com.